Add `ix_ele` to `bunch1`

electronsandstuff opened the pull request:
This PR adds the coordinate `ix_ele` as an option to `bunch1` which will allow users to get information about particle loss from tao. Also added was some comments that I made while reading the code and understanding it.

ChristopherMayes commented:
`pytao/interface_commands.py` is auto-generated, so you don't want to edit it. You'll want to look at `generate_interface_commands.py`.

electronsandstuff commented:
Ok, I have now made the change in the description of `bunch1` in bmad and have an open PR for it (bmad-sim/bmad-ecosystem#994). The changes in this PR are now the following.

- `generate_interface_commands.py` was run on the latest JSON coming from bmad which includes the updated datatypes for `bunch1`.
- I tested the new command using a script which needs `ix_ele` from the `bunch1` command.
- Added a small header to the top of autogenerated files saying that changes there will get overwritten.
- Moved a docstring I added to `pytao/interface_commands.py` to `generate_interface_commands.py` since it already exists.
